src/main/java/de/tum/in/www1/artemis/service/connectors/bitbucket/dto/BitbucketBranchProtectionDTO.java

```diff
@@ -9,6 +9,12 @@
 
     private MatcherDTO matcher;
 
+    /**
+     * needed for Jackson
+     */
+    public BitbucketBranchProtectionDTO() {
```

Comment:
@derLalla I believe you could prevent these warning if you put the documentation inside the (empty) constructor instead of above, in case you are interested in changing this.

Otherwise just resolve it again.